README.md

```diff
@@ -39,9 +38,9 @@ ran against this extention library. If you are porting your code to this codebas
 Some of the major differences between this library and Numenta's extension library are the following:
 
  * Support for Python 3 and Python 2.7 (Only Python 3 under windows)
- * Support for Linux, OSx, and Windows MS Visual Studio 2017
+ * Cross Platform Support for Linux, OSx, and Windows MS Visual Studio 2017
  * Support for C++11 through C++17
- * Replaced SWIG with PyBind11 for Python interface.
+ * Improved python interface, by replaced SWIG with PyBind11.
```

Comment:
typo "by replaced"

Reply:
Please feel free to push to this branch.  There are a lot of typos in this file... I need to go to work now so I will work on this more later.